cms/search/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class SearchConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```

Comment:
```suggestion
    default_auto_field = "django.db.models.AutoField"
```

at most. Given you don't have any special config, I'd say `apps.py` is not needed here

Reply:
Removed!